taxi/admin.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from .models import Driver, Car, Manufacturer
+
+from .models import Car, Driver, Manufacturer
```

Comment:
The import statement for local application models should be grouped separately from the Django imports. Consider reordering the imports to follow the convention: standard library imports, third-party imports, and then local application imports.